package.json

```diff
@@ -10,7 +10,11 @@
   "scripts": {
     "debug": "node-debug --hidden node_modules _mocha --watch",
     "debug-ide": "./bin/debug-ide",
-    "test": "mocha -R mocha-multi --reporter-options nyan=-,json=.results.json"
+    "copy-git-hooks": "./bin/copy-hooks",
+    "test": "mocha -R mocha-multi --require test/root.js --reporter-options nyan=-,json=.results.json",
+    "bundle": "browserify index.js -t babelify --exclude react/addons --exclude react/lib/ExecutionEnvironment --exclude react/lib/ReactContext -o bundle.js",
+    "bundle:test": "browserify test/index-test.js -t babelify --exclude react/addons --exclude react/lib/ExecutionEnvironment --exclude react/lib/ReactContext -o browser-test-bundle.js",
```

Comment:
They seem to be reliable _for now_, but like you mentioned, they like moving things around. When it gets to that point, we can run a script to update the `bundle:test` script — I already wrote a script for updating a set of repositories so we can retool that if it ever gets to that point.

Come to think of it, if they do decide to update it, I think the exclusions won't break anything. From what I've read in the Enzyme issues/docs, it seems like they're going to include this stuff in other npm packages on the top level, so these exclusions shouldn't matter much.